infrastructure/load-balancers.yaml

```diff
@@ -67,3 +95,13 @@ Outputs:
     Listener:
         Description: A reference to a port 80 listener
         Value: !Ref LoadBalancerListener
+
+    PublicAlbDnsName:
+        Value:
+            !GetAtt LoadBalancer.DNSName
+    PublicAlbHostname:
+        Value:
+            !If [ NoSslCertificate, !Join [ '', [ 'http://', !GetAtt LoadBalancer.DNSName ] ], !Join [ '', [ 'https://', !GetAtt LoadBalancer.DNSName ] ] ]
+    SslCertificate:
+        Value:
+            !If [ SslCertificate, True, False ]
```

Comment:
Since `SslCertificate` is false, we're saying False for a SslCertificate value here.